docs/Users_Guide/glossary.rst

```diff
@@ -5424,3 +5454,99 @@ METplus Configuration Glossary
 
    PREPBUFR_DATA_DIR
      .. warning:: **DEPRECATED:** Please use :term:`PB2NC_INPUT_DIR` instead.
+
+   GEN_VX_MASK_INPUT_DIR
+     Directory containing input data to GenVxMask. This variable is optional because you can specify a
+     full path to the input files using :term:`GEN_VX_MASK_INPUT_TEMPLATE`.
+
+     | *Used by:* GenVxMask
+     | *Family:* [dir]
+     | *Default:* None
+
+   GEN_VX_MASK_INPUT_TEMPLATE
+     Filename template of the input grid used by GenVxMask. This can be an input filename or a grid definition.
+     See also :term:`GEN_VX_MASK_INPUT_DIR`.
+
+     | *Used by:* GenVxMask
+     | *Family:* [filename_templates]
+     | *Default:* None
+
+   GEN_VX_MASK_INPUT_MASK_DIR
+     Directory containing mask data used by GenVxMask. This variable is optional because you can specify the
+     full path to the input files using :term:`GEN_VX_MASK_INPUT_MASK_TEMPLATE`.
+
+     | *Used by:* GenVxMask
+     | *Family:* [dir]
+     | *Default:* None
+
+   GEN_VX_MASK_INPUT_MASK_TEMPLATE
+     Filename template of the mask files used by GenVxMask. This can be a list of files or grids separated
+     by commas to apply to the input grid. The wrapper will call GenVxMask one for each item in the list, passing
```

Comment:
"The wrapper will call GenVxMask one for each item in the list"; should be "once"

Reply:
Thanks for catching that